index.html

```diff
@@ -169,8 +193,10 @@ <h3>skills</h3>
           <div><img class="dropdown" src="icon-images/Icon - Chevron - Right - Larger.png" alt="right dropdown"></div>
         </li>
       </ul>
+    </div>
     </section>
-
+    </div>
+      
     <section class="forms-section">
       <form action="https://formspree.io/f/mdojpdrk" method="post">
         <fieldset>
```

Comment:
- [ Optional ] For Minimum mobile screen size should be 375px, Your  "contact me" section is not aligned correctly,Kindly adjust it.

Your contact me section at 375px look likes this : 
  
![Capture](https://user-images.githubusercontent.com/93598189/199969456-902adfd6-1dc9-4e09-976c-87f4bf32d23f.PNG)

The Figma design "Contact section" Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/199969809-4b3177fc-f622-464d-926b-d30e6d2126d3.PNG)